AWSElasticLoadBalancingTests/AWSElasticLoadBalancingTests.m

```diff
@@ -44,60 +44,19 @@ - (void)testClockSkewELB {
 
     XCTAssertFalse([NSDate aws_getRuntimeClockSkew], @"current RunTimeClockSkew is not zero!");
     [AWSTestUtility setMockDate:[NSDate dateWithTimeIntervalSince1970:3600]];
+
     AWSElasticLoadBalancing *elb = [AWSElasticLoadBalancing defaultElasticLoadBalancing];
     XCTAssertNotNil(elb);
 
-    AWSElasticLoadBalancingDescribeAccessPointsInput *describeAccessPointsInput = [AWSElasticLoadBalancingDescribeAccessPointsInput new];
-    [[[elb describeLoadBalancers:describeAccessPointsInput] continueWithBlock:^id(AWSTask *task) {
-        if (task.error) {
-            XCTFail(@"Error: [%@]", task.error);
-        }
-
-        if (task.result) {
-            XCTAssertTrue([task.result isKindOfClass:[AWSElasticLoadBalancingDescribeAccessPointsOutput class]]);
-            AWSElasticLoadBalancingDescribeAccessPointsOutput *describeAccessPointsOutput = task.result;
-            XCTAssertNotNil(describeAccessPointsOutput.loadBalancerDescriptions, @"loadBalancerDescriptions Array should not be nil");
-        }
+    AWSElasticLoadBalancingDescribeAccountLimitsInput *input = [AWSElasticLoadBalancingDescribeAccountLimitsInput new];
 
+    [[[elb describeAccountLimits:input] continueWithBlock:^id(AWSTask *task) {
+        XCTAssertNil(task.error);
+        XCTAssertNotNil(task.result);
```

Comment:
are these hand written tests which is why it failed after the model breaking changes?

Reply:
Correct. We do not codegen integration tests. The most basic integration test, which is demonstrated here, is a clock skew test that picks some arbitrary API call (usually a read, list, or describe call) and ensures that it authenticates.